berkeley-function-call-leaderboard/bfcl/eval_checker/multi_turn_eval/func_source_code/travel_booking.py

```diff
@@ -570,7 +589,7 @@ def compute_exchange_rate(
 
         """
         exchange_rates = {
-            ("USD", "RMS"): 7,
+            ("USD", "RMB"): 7,
```

Comment:
Should be `RMS` here.

Reply:
RMB is a currency, but not RMS? So, maybe the right thing to do here is RMB?

Reply:
[Here](https://github.com/ShishirPatil/gorilla/blob/956fa0df849c425ade127bdd63bd51e599790831/berkeley-function-call-leaderboard/bfcl/eval_checker/multi_turn_eval/func_source_code/travel_booking.py#L672), `RMS` is a 'fake' airport code that stands for `Rivermist`. The inclusion of fake airport is to avoid model retrieving the airport code from its memory instead of using the relative functions.

Reply:
It seems from the variable and function name that RMB should be a currency. If you believe it should still be RMS then it would be good to fix the ground truths that call this function with 'RMB'. I listed a few above.

Reply:
This should be RMB. Resolved offline.